provider/dynamodb.go

```diff
@@ -364,6 +371,10 @@ func (store *dynamodbOnlineStore) CheckHealth() (bool, error) {
 	return true, nil
 }
 
+func (store dynamodbOnlineStore) Delete(location pl.Location) error {
+	return fferr.NewInternalErrorf("delete not implemented")
```

Comment:
Leave a comment explaining its not implemented just because there isn't an idea of a dyanmoDB location